pkg/ignition/ignition.go

```diff
@@ -12,22 +12,48 @@ import (
 	"github.com/pkg/errors"
 )
 
+type OSInterface interface {
```

Comment:
Wasn't that added on another PR? Or is it just based on it:/

Reply:
that's a different pkg

Reply:
now using the generic fileutil.OSInterface